assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -336,3 +338,7 @@ function LegalNotice() {
 // ----- Exports ----- //
 
 export default connect(mapStateToProps, mapDispatchToProps)(DirectDebitForm);
+
+LegalNotice.defaultProps = {
+  countryGroupId: 'GBPCountries',
+};
```

Comment:
As `countryGroupId` is a required prop of `DirectDebitForm`, it will always exist when passed through to `LegalNotice`, so I don't think this `defaultProps` declaration will be used. One option is to delete it and accept that `DirectDebitForm` will always take a `countryGroupId`. Alternatively you could decide that this prop is optional for `DirectDebitForm` itself, and migrate `defaultProps` to that component:

```jsx
DirectDebitForm.defaultProps = {
  countryGroupId: 'GBPCountries',
};
```

Minor side note, we try to keep `export` statements at the very bottom of the file by convention (with the exception of files where we have an `export default` in line with the component function for some reason, perhaps that's not a good exception...). In this case ideally the `export default connect...` line would sit at the bottom of the file, as in the [`FeatureList`](https://github.com/guardian/support-frontend/blob/master/assets/components/featureList/featureList.jsx#L90) component.